demisto_sdk/commands/validate/file_validator.py

```diff
@@ -437,22 +440,23 @@ def validate_added_files(self, added_files):  # noqa: C901
                     self._is_valid = False
 
             # incident fields and indicator fields are using the same scheme.
-            elif checked_type(file_path, JSON_INDICATOR_AND_INCIDENT_FIELDS):
+            elif checked_type(file_path, JSON_INDICATOR_AND_INCIDENT_FIELDS) or \
+                    file_type in ('incidentfield', 'indicatorfield'):
                 incident_field_validator = IncidentFieldValidator(structure_validator)
-                if not incident_field_validator.is_valid_file():
+                if not incident_field_validator.is_valid_file(validate_rn=False):
```

Comment:
if we are in git mode, we should validate release notes.
`validate_rn=not file_type`
`file_type` is None -> not using general path file -> should check RN
`file_type` is not None -> using general path file -> should not check RN

relevant for pre-commit hooks, in the build we have another step that will fail on this check.